api_tests/users/serializers/test_serializers.py

```diff
@@ -58,3 +194,36 @@ def test_user_serializer(self, user):
         assert 'institutions' in relationships
         assert 'preprints' in relationships
         assert 'registrations' in relationships
+
+    def test_related_counts_equal_related_views(self,
+                                                request,
+                                                field_name,
+                                                expected_count,
+                                                user,
+                                                user_without_nodes,
+                                                project,
+                                                public_project,
+                                                deleted_project,
+                                                registration,
+                                                private_registration,
+                                                withdrawn_registration,
+                                                preprint,
+                                                private_preprint,
+                                                withdrawn_preprint,
+                                                unpublished_preprint,  # not in the view/related counts by default
+                                                deleted_preprint):
```

Comment:
This is a lot of fixtures, but I thought it would be best to include them all in case there's some weird interactions, it also allows new cases to be added easily without writing new tests.